libp2p/NodeTable.h

```diff
@@ -275,6 +288,7 @@ class NodeTable : UDPSocketEvents
         return dev::p2p::isAllowedEndpoint(m_allowLocalDiscovery, _endpointToCheck);
     }
 
+    bool m_enabled;                                                 /// Is discovery enabled?
```

Comment:
this doesn't look used?

Reply:
Ah yes artifact of an earlier change that I forgot to remove. Will remove it